src/models/reposyncstate.ts

```diff
@@ -210,5 +213,6 @@ RepoSyncState.init({
 	syncCompletedAt: DATE,
 	config: JSON,
 	createdAt: DATE,
-	updatedAt: DATE
+	updatedAt: DATE,
+	commitFrom: DATE
```

Comment:
/nit just for code readability, maybe move this line above to after the `deploymentCursor`, so it is like
```
xxxStatus
xxxStatus
xxxStatus
xxxCursor
xxxCursor
xxxCursor
xxxFrom
xxxFrom
xxxFrom
```